src/lib/integration_test_cloud_engine/stack_driver_log_engine.ml

```diff
@@ -598,8 +598,8 @@ let rec pull_subscription_in_background ~logger ~subscription_name
       let%bind logs = Subscription.pull subscription in
       Malleable_error.List.map logs ~f:parse_subscription)
   in
-  [%log debug] "Pulling subscription $subscription_name"
-    ~metadata:[("subscription_name", `String subscription_name)] ;
+  (* [%log debug] "Pulling subscription $subscription_name"
```

Comment:
Don't comment this please. If you want to get rid of this from your output, just pipe through `logproc -f '.level != "Debug"'` instead.

Reply:
sorry forgot to comment this back in